akka-actor/src/main/scala/akka/routing/RouterConfig.scala

```diff
@@ -140,7 +140,10 @@ abstract class GroupBase extends Group {
  */
 trait Group extends RouterConfig {
 
-  def paths: immutable.Iterable[String]
+  @deprecated("Implement paths with ActorSystem parameter instead", "2.4")
+  def paths: immutable.Iterable[String] = null
+
+  def paths(system: ActorSystem): immutable.Iterable[String]
```

Comment:
oh, great that the same trick as the other ticket in this area works for this ticket as well :)